lib/commands.js

```diff
@@ -329,33 +333,24 @@ function _optimizeJS(callback) {
   async.auto({
     templates: callback => _optimizeAngularTemplates(callback),
     buildImportAll: callback => packages.buildImportAllModule(callback),
-    buildAliases: callback => _buildAliases(callback),
-    webpack: ['templates', 'buildImportAll', 'buildAliases',
-      (results, callback) => {
-      brWebpack.optimize({
-        main: [
+    optimize: ['templates', 'buildImportAll', (results, callback) => {
+      const options = {
+        input: [
           bedrock.config.views.angular.optimize.templates.out,
           bedrock.config.views.system.paths.mainModule
         ],
-        configs: [
-          ...results.buildAliases,
-          {
-            resolve: {
-              modules: [
-                path.dirname(bedrock.config.views.system.paths.importAll)
-              ]
-            }
-          }
-        ]
-      }, callback);
+        output: bedrock.config.views.system.paths.mainMin
+      }
+      // FIXME: document this better:
+      // options:
+      //   input: array of input files to optimize together
+      //   output: filename to output
+      bedrock.events.emit('bedrock.optimize.run', options, callback);
```

Comment:
How about `bedrock-views.optimize.run`?  Plan ahead in case it's called from code vs the cli.